From 722d9b8bb5b6673b00d4906dd9198353d21b9aa4 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Tue, 4 Sep 2012 10:22:00 +0800
Subject: [PATCH 071/602] ENGR00222899 init.rc: fix the /data mount option
 error

The noauto_da_alloc and errors=panic are both option
not flag. In init code, the mount command should be:
mount <type> <device> <path> <flags ...> <options>

The options must be separated by ',' not ' '.

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 imx6/sabresd/init.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/imx6/sabresd/init.rc b/imx6/sabresd/init.rc
index bb275e4..e77d869 100755
--- a/imx6/sabresd/init.rc
+++ b/imx6/sabresd/init.rc
@@ -132,6 +132,6 @@ on fs
 # mount ext4 partitions
     mount ext4 /dev/block/mmcblk0p5 /system
     mount ext4 /dev/block/mmcblk0p5 /system ro remount
-    mount ext4 /dev/block/mmcblk0p7 /data nosuid nodev nodiratime noatime noauto_da_alloc errors=panic
+    mount ext4 /dev/block/mmcblk0p7 /data nosuid nodev nodiratime noatime noauto_da_alloc,errors=panic
     mount ext4 /dev/block/mmcblk0p6 /cache nosuid nodev
     mount ext4 /dev/block/mmcblk0p8 /device ro nosuid nodev
-- 
1.8.0

